App.tsx

```
@@ -1,101 +1,126 @@
+        drawerActiveBackgroundColor: '#334155',
+      }}
       initialRouteName="Dashboard"
     >
-      <Drawer.Screen name="Dashboard">
-        {(props) => <DashboardScreen {...props} onLogout={handleLogout} />}
-      </Drawer.Screen>
-      
-      {/* --- NEW MENU ITEMS --- */}
+      <Drawer.Screen
+        name="Dashboard"
+        children={(props) => <DashboardScreen {...props} onLogout={handleLogout} />}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)